staff/fede-gomez/test-sass/style.css.map

```diff
@@ -1 +1,9 @@
-{"version":3,"sourceRoot":"","sources":[],"names":[],"mappings":"","file":"style.css"}
\ No newline at end of file
+{
```

Comment:
This file should not be here.

Reply:
I don´t want to change this in case I delete Fedes work...